lib/ecto/multi.ex

```diff
@@ -619,6 +619,31 @@ defmodule Ecto.Multi do
     add_operation(multi, name, {:put, value})
   end
 
+  @doc """
+  Inspects results from a Multi
+
+  By default, the name is shown as a label to the inspect, custom labels are
+  supported through the `IO.inspect/2` `label` option.
+
+  ## Options
+    All options for IO.inspect/2 are supported.
+
+    * `:only` - A list of fields to inspect, will print the entire map by
+      default.
+
+  ## Example
+
+      Ecto.Multi.new()
+      |> Ecto.Multi.insert(:person, changeset)
+      |> Ecto.Multi.inspect()
+      |> MyApp.Repo.transaction()
+
+  """
+  @spec inspect(t, name, keyword()) :: t
+  def inspect(multi, name, opts \\ []) do
```

Comment:
Let's remove the `name` as argument. Instead, you can set internally it to always be `:inspect` (we do similar for `merge`). Then we show no label by default and leave it to be customized via the label option.